test/rfq_blockchain_utils_test.ts

```diff
@@ -312,7 +314,8 @@ describe(SUITE_NAME, () => {
             expect(result.to).to.equal(zeroEx.address);
         });
     });
-    describe('signTransactionRequestAsync', () => {
+    // Unskip this test if/when ganache 7.0.0, with EIP1559 support, is released.
+    describe.skip('signTransactionRequestAsync', () => {
```

Comment:
this might actually be a good test to see if our code can handle both EIP-1559 chains and legacy chains. In other words, we treat ganache as a legacy chain (until we update to 7.0.0)